README.md

```diff
@@ -184,13 +184,17 @@ syslog-ng packages are released for the following distribution versions (x86-64)
     wget -qO - https://ose-repo.syslog-ng.com/apt/syslog-ng-ose-pub.asc | sudo apt-key add -
     ```
 
-2. Add the repository containing the latest build of syslog-ng to the APT sources. For example, stable releases on Ubuntu 22.04:
+2. Add the repository containing the latest build of syslog-ng to the APT sources. For example, stable releases on Ubuntu 24.04:
 
     ``` shell
-    echo "deb https://ose-repo.syslog-ng.com/apt/ stable ubuntu-jammy" | sudo tee -a /etc/apt/sources.list.d/syslog-ng-ose.list
+    echo "deb https://ose-repo.syslog-ng.com/apt/ stable ubuntu-noble" | sudo tee -a /etc/apt/sources.list.d/syslog-ng-ose.list
     ```
 
-3. Run `apt update`
+3. Now install syslog-ng:
```

Comment:
Please keep the update step as well as a separate step as it is important

Even though the install step is already provided above, and this part is just about explaining some prerequisites, you can add the install step if you wish, but please do that as a separate step

Reply:
Though markdown possibly handles the duplicated numbers, but please use the next one in the row here

3. -> 4.

for better readability
thanks!